app/views/sign_up/registrations/new.html.erb

```diff
@@ -17,6 +17,22 @@
 
 <%= render PageHeadingComponent.new.with_content(t('headings.create_account_new_users')) %>
 
+<% if IdentityConfig.store.account_creation_device_profiling_enabled %>
+  <% if @threat_metrix_variables[:threatmetrix_session_id].present? %>
```

Comment:
(Suggestion; non-blocking) Maybe this could be refactored into a partial and shared with the IDV SSN views